windows/utils/win_enable_vmtools_log.yml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023-2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Enable debug logging for vmtools within Windows guest OS
+# See https://knowledge.broadcom.com/external/article?legacyId=1007873 for details.
+#
+- name: "Check and create VMware Tools config file"
+  include_tasks: win_check_create_vmtools_conf.yml
+
+- name: "Enable vss log in VMware Tools config file"
```

Comment:
Here is not enabling vss log, but enabling VMware Tools debug logging.
And there is a common task `update_ini_style_file.yml` can update ini file.

Reply:
Keep consistent with file utils/win_enable_vss_log.yml.